Example/Tests/CryptoTests.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CryptoTests.swift
+//  uPortSDK_Tests
+//
+//  Created by Aldi Gjoka on 12/5/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import Quick
+import Nimble
```

Comment:
This test is fine using plain XCTest; no need for external Quick & Nimble.  I believe that as little as possible external libraries should be used (because it keeps project dependencies low and makes our SDK easier/wider accessible).  Could you please rewrite?